common/lib/xmodule/xmodule/mongo_utils.py

```diff
@@ -58,9 +58,14 @@ def connect_to_mongodb(
             wait_time=retry_wait_time
         )
 
+    # default the authSource to be whatever db we are connecting to (for backwards compatiblity)
```

Comment:
```suggestion
   # Allow customizing the authSource to be more compatible with different MongoDB providers.
```

Please make the comment more readable and focus on the `why` rather than the `what`. If the suggestion above doesn't sound good, please feel free to suggest an alternative one.